src/arrays.js

```diff
@@ -1,59 +1,101 @@
+
+
 export const getNthElement = (index, array) => {
-  // your code here
+  if (index + 1 > array.length){
+    return array[index - array.length];
+  }
+  return array[index];
```

Comment:
I like to have a line break before a `return` statement (unless it's the only line of code in a block) so it's clearer.